onnxruntime/test/optimizer/resnet50_fusion_test.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include <vector>
+#include "core/graph/graph.h"
+#include "core/optimizer/conv_activation_fusion.h"
+#include "core/optimizer/conv_add_fusion.h"
+#include "core/optimizer/conv_add_act_fusion.h"
+#include "gtest/gtest.h"
+#include "graph_transform_test_builder.h"
+#include "test/test_environment.h"
+#include "test/util/include/asserts.h"
+
+namespace onnxruntime {
+namespace test {
+#if defined(ORT_RUN_EXTERNAL_ONNX_TESTS) && !defined(DISABLE_CONTRIB_OPS)
+class ResNet50FusionTests : public ::testing::Test {
+ protected:
+  ResNet50FusionTests() : logger(DefaultLoggingManager().CreateLogger("ResNet50FusionTest")) {
+  }
+  std::unique_ptr<logging::Logger> logger;
+
+  void SetUp() override {
+    if(Model::Load(fp32_model_path, fp32_model, nullptr, *logger)!=Status::OK()){
+      GTEST_SKIP() << "Failed to load model: " << fp32_model_path;
+    }
+    if (Model::Load(fp16_model_path, fp16_model, nullptr, *logger) != Status::OK()) {
+      GTEST_SKIP() << "Failed to load model: " << fp16_model_path;
+    }
+  }
+  std::basic_string<ORTCHAR_T>  fp32_model_path = ORT_TSTR("../models/opset10/Resnet50_Fusion_Testing/resnet50.onnx");
+  std::shared_ptr<Model> fp32_model;
+  std::basic_string<ORTCHAR_T> fp16_model_path = ORT_TSTR("../models/opset10/Resnet50_Fusion_Testing/resnet50.fp16.onnx");
```

Comment:
I just move the file from "Resnet50_Fusion_Testing/resnet50.fp16.onnx" to "Resnet50_Fusion_Testing_fp16/resnet50.fp16.onnx", because WinML's test code has some restrictions that one folder can only have one model.

Reply:
Please update this line of code accordingly.